Sky Atlas/Sky Atlas.html

```diff
@@ -26054,7 +26125,7 @@ <h1>Agent Scope Database</h1>
             <pre><code>
               foreignController.redeemERC4626(address(token), token.balanceOf(address(proxy)))
```

Comment:
```suggestion
              foreignController.redeemERC4626(address(token), token.balanceOf(address(proxy))
```

This is a deviation from the forum post, but is it possible to remove this extra parenthesis?